api/ephemeral-access/v1alpha1/accessrequest_types.go

```diff
@@ -25,10 +25,13 @@ import (
 
 // Status defines the different stages a given access request can be
 // at a given time.
-// +kubebuilder:validation:Enum=requested;granted;expired;denied;invalid
+// +kubebuilder:validation:Enum=initiated;requested;granted;expired;denied;invalid
 type Status string
 
 const (
+	// InitiatedStatus is the stage that defines the access request as initiated
+	InitiatedStatus Status = "initiated"
```

Comment:
What is the difference between initiated and requested? What would cause an access request to be initiated, but not yet requested?

Reply:
With the plugin introduction we now need to differentiate between Initiated and Requested.
Initiated: The initial status that every AccessRequest will have.
Requested: The status used to determine that the AccessRequest is waiting for an external system.

Reply:
Do we have request state expiry? what happen when user discard the request and wanrt to generate a new one later